### PR TITLE
Hawkey tests are passing

### PR DESCRIPTION
**Pull request by cibo94**

### CODE REVIEW

**Comment by jsilhan** [comment]
append there a commit that fixes Python plugin too, please. I will merge it once the issue is completely solved

File: rpg/plugins/lang/python.py [hunk 1]
```diff
@@ -15,3 +17,16 @@ def patched(self, project_dir, spec, sack):
         for _, mod in mod.modules.items():
             if mod.__file__ and mod.__file__.startswith("/usr/lib"):
                 spec.required_files.add(mod.__file__)
+
+    def installed(self, project_dir, spec, sack):
+        if list(project_dir.glob('**/*.py')):
+            compile_python = Command(
+                [
+                    'python -c \'import compileall;' +
```
Comment (jsilhan) [inline rpg/plugins/lang/python.py]: shouldn't we differentiate between `python2 -c \'import compileall;` and `python3 -c \'import compileall;`?

**Comment by jsilhan** [comment]
Miro, tell me once this PR is fully finished and ready to be merged?

File: rpg/plugins/misc/find_file.py [hunk 2]
```diff
@@ -1,22 +1,10 @@
...
-                self.exclude[re.sub(r"[^/]*$", "",
-                                    "/" + str(item.relative_to(project_dir))) +
-                             "__pycache__/"] = True
-        sorted_files = sorted(self.files +
-                              [(key, r'%exclude', None)
-                               for key in self.exclude],
-                              key=lambda e: e[0])
-        for one_file in sorted_files:
-            spec.files.add(one_file)
+            if item.is_file():
```
Comment (jsilhan) [inline rpg/plugins/misc/find_file.py]: great, this is how it should look like. The previous code was python specific and have should not been there. :thumbsup:

**Comment by cibo94** [comment]
all tests are passing now.

**Comment by cibo94** [comment]
c6ec154 is needed by libsolv to pass. I also resolve issue #296 with it, because we were testing c plugin for meaningless include files for instance:
this was in C test file: ```#include <stdio.h>``` so we should test if ```/usr/include/stdio.h``` was found but we were testing a lot of system dependent files like ```/usr/lib/gcc/x86_64-redhat-linux/5.1.1/include/stdarg.h``` and so on.

**Comment by jsilhan** [comment]
Merged, thanks

**Pull request closed without merging**